### ISSUE TITLE
Maintained?

### ISSUE THREAD

**Issue by amaanq** [eonpatapon/tree-sitter-cue#11]
Hi, I'd like to be a contributor/maintainer of this repo so that I can add it to upstream nvim-treesitter. It seems to be a bit dead atm, so I'll maintain a fork unless a current maintainer is active or I am added as a contributor. Let me know :slightly_smiling_face: 

Thanks,
-Amaan

**Comment by eonpatapon** [comment]
Hi, why do you say it is dead ?

Did you see parsing bugs ?

**Comment by eonpatapon** [comment]
It would be great to include it in nvim-treesitter

**Comment by amaanq** [comment]
Yes, there are quite some bugs but don't worry I will fix them. I will have a PR coming soon detailing more.

![image](https://user-images.githubusercontent.com/29718261/215452511-5a25ff42-980e-4d96-a0a5-ae8c9c03a62c.png)

**Comment by amaanq** [comment]
Sorry about the delay, I forgot about it for a bit, opening a PR now!
